microovn/src/charm.py

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env python3
-import logging, ops, os
+import json
+import logging
+import ops
```

Comment:
Among the imports, the `ops` package stands out as the only one not being part of the standard Python library.

Would it make sense to keep those in a separate group of imports after the standard library imports?